packages/web-react/src/components/Modal/Modal.stories.tsx

```diff
@@ -1,18 +1,78 @@
-import { ComponentMeta } from '@storybook/react';
-import Modal from './Modal';
+import React, { useState } from 'react';
+import { Markdown } from '@storybook/blocks';
+import type { Meta, StoryObj } from '@storybook/react';
 
-export default {
+import { SpiritModalProps } from '../../types';
+import { Button } from '../Button';
+import ReadMe from './README.md?raw';
+import { Modal, ModalHeader, ModalDialog, ModalBody, ModalFooter } from '.';
+
+const meta: Meta<typeof Modal> = {
   title: 'Components/Modal',
   component: Modal,
   parameters: {
     docs: {
-      description: {
-        component: 'Add dialogs to your site for lightboxes, user notifications, or completely custom content.',
+      page: () => <Markdown>{ReadMe}</Markdown>,
+    },
+  },
+  argTypes: {
```

Comment:
Can be args and argTypes in separate files, please?

Reply:
They can, but it doesn't make sense, because they are now used only once in one file. If they are used in more (currently in Button and ButtonLink, I keep them in separate files.